Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -660,3 +673,23 @@ enum KituraWebSocketUpgradeError: Error {
     // Unknown upgrade error
     case unknownUpgradeError
 }
+
+class Atomic<A> {
```

Comment:
you could use [NIO's Atomic<T>](https://apple.github.io/swift-nio/docs/current/NIOConcurrencyHelpers/Classes/Atomic.html) which actually is atomic and doesn't require a lock. Just add `NIOConcurrencyHelpers` to `Package.swift` and `import NIOConcurrencyHelpers`